docs/graphql/manual/troubleshooting/index.rst

```diff
@@ -56,21 +56,21 @@ Solution
 Error: field already exists
 ---------------------------
 
-When a relationship is created using Hasura GraphQL Engine, it creates a special field with the relationship name
+When a relationship is created using the Hasura GraphQL engine, it creates a special field with the relationship name
```

Comment:
I noticed a few files still using Engine (with capital E). Maybe a global search for `GraphQL Engine` and replace to `GraphQL engine` should make it consistent throughout